packages/neuron-ui/src/contexts/transfer.ts

```diff
@@ -0,0 +1,12 @@
+import { createContext } from 'react'
```

Comment:
We don't need a Transfer context.

We don't need context for every feature to add. :P

Reply:
ok